src/main/java/nextstep/ladder/domain/Column.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.ladder.domain;
+
+import java.util.Objects;
+
+public class Column {
+  private int column;
```

Comment:
```suggestion
  private final int column;
```
인스턴스 변수는 기본은 final로 구현하고 값이 변경되는 상황이 생겼을 때 final을 제거하는 습관을 들이는 것은 어떨까?